### DISCUSSION TITLE
关于一个网盘挂载到多个alist的疑问。

### DISCUSSION THREAD

**Discussion by StartMoney** [AlistGo/alist#4995] [Q&A]
阿里云盘1
天翼云盘1
alist1.com
alist2.com

1、先说阿里云盘：
当阿里云盘1同时挂载到两个alist网站服务器上时，aliyun open认证是不是会出问题？
如果两个alist网站用同一个open接口，那么token授权码是不是该用一个？
如果用一个，那token更新的refresh token岂不是会不同步，或者说其中一个会失效，然后死循环，1跟新完2失效，2又更新，1又失效。。。。
如果两个alist网站不使用同一个open接口，我不清楚alist官方提供的三个接口是不是共享了token授权码的，如果共享了，是不是跟上面一个情况了。如果没共享，应该就可以避免相互挤兑失效的问题了。
所以，我想问的是：我要在两个alist网站上同时挂载同一个阿里云盘1时，需要怎么做才不会交叉影响，致使认证失效？

2、再说天翼云盘：
天翼云盘是保存用户名密码进两个alist网站的，相当于同一个账号在不同终端同时登录天翼云盘服务器，后认证的服务器会把前认证的服务器挤下线吗？如果会，又进入死循环了。如果不会，那天翼云多终端登录同一账号的个数是多少呢？

**Comment by foxxorcat** [comment]
这就是为什么说aliyun可能存在一个死循环错误推荐使用aliyun open了。并且每个网盘认证不一样，现在不出问题不代表后面不出问题。如果可以还是尽量避免这个问题。
1. aliyun open的token刷新不会造成挤下线
2. 天翼云盘属于登录操作，除非服务器限制登录数量

**Reply by StartMoney** [reply]
感谢回复。现在alist官方的认证接口共享数据没有呢？我两个站点用同一个token好还是分开token?

**Comment by anwen-anyi** [comment]
一个阿里云盘open账号挂载多次没有问题的
至于你拿刚刚获取到token同时挂载多个只要能挂载就是没问题的
至于验证OA授权的都是一样的，api.nn.ci好像是源站,另买两个应该是反向代理了 所以数据是一样的通用

**Reply by StartMoney** [reply]
感谢回复。刚查了，在阿里云盘的个人管理里面有登录设备管理。10个设备登录。
